src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultTag/Components/defaultTag.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { InputTypes } from '../types';
+import { StyledTag1, StyledTag2, StyledTag3, StyledTag4, StyledIcon } from '../styles';
+import { AppIcon } from '../../../../types/iconsTypes/iconEnums';
+
+export const DefaultTag1 = ({ label }: InputTypes) => (
+  <React.Fragment>
+    <span style={StyledTag1}>{ label }</span>
```

Comment:
Please avoid` inline-styles` use styled components like we are using in rest of the application.

Reply:
Done